test/feature/Modules/Error_ImportType.module.js

```diff
@@ -0,0 +1,4 @@
+// Options: --types
+// Error: :4:14: 'missing' is not exported by 'test/feature/Modules/resources/default-and-named-type.js'
```

Comment:
I looked at it and it is not obvious. The way flow treats exports as types requires program flow analysis.

```js
class C {}
var x = C;
export {x as y};
```

Here the module exports y as a function but also as a type.